src/components/ViewResponses/Responses.js

```diff
@@ -0,0 +1,43 @@
+import { Card, Typography } from "antd";
+
+const Response = (props) => {
+  let { allResponses, userInfo } = props;
+
+  const { Title } = Typography;
+  allResponses && allResponses.length !== 0 && (
```

Comment:
allResponses?.length !==0

Reply:
this would be wrong since if allresponses is null, it would return null which would make the output as null which is !==0 which would execute this statement, which is not the intended effect